content/articles/getting-started-with-python-plug-ins-for-gimp/index.md

```diff
@@ -0,0 +1,158 @@
+- GIMP is an open-source picture control program that many individuals use as a feasible option in contrast to a portion of the business contributions, subsequently making it more straightforward to introduce on your gadget. You can download the most recent version of GIMP [here](http://www.gimp.org/downloads/). 
+- After an effective establishment of GIMP programming the next thing to introduce is Python on your working framework as indicated by the guidance on the python documentation. You can download the most recent version of python [here](https://www.python.org/downloads/). To affirm that you have effectively introduced python on your OS, open the command line and type 'python --version'. 
+You can likewise get GIMP-python documentation [here](http://www.gimp.org/docs/python/)
+
+### Enrolling your content
+On the Windows stage, Python records go in your client's home GIMP folder. For my situation its 
+'C:\Users\KENNEDY\AppData\Roaming\GIMP\2.10\plug-ins' the place where I have saved my `.py` record 
+- Note that framework modules are a hidden folder; thus, to get to Gimp 2.10 modules index in Windows 10 open File Explorer and enter the above path from your GIMP. 
+You can get the above path by opening your GIMP, tapping on 'edit>preferences>folders>plug-ins' 
+The figure beneath shows the way of getting the document area 
+
+![Plug-ins folder](plugins.png) 
+
+the chose path above is for '`system plug-ins`', and the other is for 'user plug-ins' duplicate the framework modules and paste it on your file explorer pathway. Make another text document in the open folder and save it as '.py' 
+On Mac and Linux frameworks, that folder is '~/.gimp-2.6/modules'. 
+The Python script records should in like manner be executable and have the Python translator on the primary line, like the standard substance declarations under.
+There are 12 arguements to the register work you might found them clarified [here](http://www.expmedia.com/content/broadening gimp-python-python-fu-modules section 2). You are needed to enlist your python script all together GIMP to add the modules in one of the GIMP menus. The following is a model
+
+``` python
+#!/usr/bin/python
+From gimpfu import * # same for every plug-ins
+def gimp_plugins(image, drawable):
+    return
+register(
+# name of the order you invoke from the command prompt
+"python_fu_resize",
+# The procedure browser displays information about the plug-in.
+"Saves the image at a maximum width and height",
+# The designer of the plug-in
+"Kennedy Mwangi",
+# The plug-copyright in's holder. 
+"Kennedy Mwangi",
+# The copyright expiration date	 
+"2021",
+# The menu label that the plug-in uses. 
+"<Image>/Image/Resize to max ",
+# The methodology parameters for the plug-in
+[
+    (PF_FILE, "infile", "Path for input file", ""), 
+     (PF_DIRNAME, "save-path", "Path for output filename", ""), 
+],
+# The results of the plug-in’s method
+[
+    menu="<image>/Resize to max../"
+],
+gimp_plugins)

```

Comment:
`From gimpfu import *` Do you think this would work?

Why is this `def gimp_plugins(image, drawable):return` empty?